pennylane/pauli/conversion.py

```diff
@@ -29,114 +29,23 @@
 
 
 # pylint: disable=too-many-branches
-def pauli_decompose(
-    matrix,
-    hide_identity=False,
-    wire_order=None,
-    pauli=False,
-    padding=False,
-) -> Union[Hamiltonian, PauliSentence]:
+def _generalized_pauli_decompose(
+    matrix, hide_identity=False, wire_order=None, pauli=False, padding=False
```

Comment:
these should probably be positional args since this is an internal-only function and it's always called with all arguments specified